config/initializers/active_model_serializers.rb

```diff
@@ -0,0 +1,166 @@
+module AMS
+  module V09
+    class Serializer < ActiveModel::Serializer
+
+       def serializable_hash(adapter_options = nil,
+                             options = {},
+                             adapter_instance = self.class.serialization_adapter_instance)
+         object.nil? ? nil : super
+       end
+    end
+
+    class ArraySerializer < ActiveModel::Serializer::CollectionSerializer
+    end
+
+    class ResourceIdentifier < ActiveModelSerializers::Adapter::JsonApi::ResourceIdentifier
+
+      def initialize(serializer, options)
+        @id   = id_for(serializer)
+        @type = type_for(serializer)
+      end
+
+    private
+
+      def id_for(serializer)
+        serializer.read_attribute_for_serialization(:id)
+      end
+
+    end
+
+    class Base < ActiveModelSerializers::Adapter::Base
+
+      def initialize(serializer, options = {})
```

Comment:
Layout/SpaceAroundEqualsInParameterDefault: Surrounding space detected in default value assignment.